stdlib/public/SDK/Dispatch/Data.swift

```diff
@@ -121,11 +121,16 @@ public struct DispatchData : RandomAccessCollection, _ObjectiveCBridgeable {
 
 	private func _copyBytesHelper(to pointer: UnsafeMutableRawPointer, from range: CountableRange<Index>) {
 		var copiedCount = 0
-		__dispatch_data_apply(__wrapped) { (_, _, ptr: UnsafeRawPointer, size: Int) in
-			let limit = Swift.min((range.endIndex - range.startIndex) - copiedCount, size)
-			memcpy(pointer + copiedCount, ptr, limit)
-			copiedCount += limit
-			return copiedCount < (range.endIndex - range.startIndex)
+		let rangeSize = range.endIndex - range.startIndex
```

Comment:
It is sort of incorrect to calculate the range size as difference between it's `startIndex` and `endIndex`. Even though `lowerBound` and `upperBound` return the same values, they should be preferred.